validphys2/src/validphys/theorycovariance/construction.py

```diff
@@ -120,7 +120,7 @@ def total_covmat_datasets(
 
 @check_correct_theory_combination
 def total_covmat_diagtheory_datasets(
-    each_dataset_results_bytheory, fivetheories: (str, type(None)) = None
+    each_dataset_results_bytheory, fivetheories
```

Comment:
For this one (and a few more in this file) you can change them to be just one line since now they are more or less small:

`def total_covmat_diagtheory_datasets(each_dataset_results_bytheory, fivetheories)`

Reply:
I believe that this has been done automatically by black now.